### PR TITLE
Add Next.js dashboard workshop

### PR DESCRIPTION
**Pull request by lachlanjc**

### CODE REVIEW

**Comment by lachlanjc** [comment]
Example rehacks:
https://nextjsdashboard--lachlanjc.repl.co/ (mine/official)
https://dashboard.smd21.repl.co/
https://lightpinkvitalcallback--carolineshen.repl.co/
https://fluidreasonabledrawing.isabelallerheil.repl.co/
https://sup-peeps--jackrabbit.repl.co/

File: workshops/nextjs_dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+        )}
+      </div>
+    )
+  }
+}
+
+export default Weather
+```
+
+On the `const url` line, you'll notice we're using this other URL with `cors-anywhere` in it. This is an inelegant hack to get our site working: Dark Sky only wants you to make "proper" requests from your servers, but since we just want to get this working in our browser, we're using this other utility.
```
Comment (MaxWofford) [inline workshops/nextjs_dashboard/README.md]: > we're using this other utility

Use styled apostrophes in markdown

File: workshops/nextjs_dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+        )}
+      </div>
+    )
+  }
+}
+
+export default News
+```
+
+You'll need to use the component though :) So head back to `pages/index.js`, and add `import News from '../components/News'` near the top and add the component to the body. Like this:
```
Comment (MaxWofford) [inline workshops/nextjs_dashboard/README.md]: > You'll

Same for all the other instances of apostrophes in the PR

File: workshops/nextjs_dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+      Powered by <a href="https://newsapi.org/">NewsAPI</a> and{' '}
+      <a href="https://darksky.net/poweredby/">Dark Sky</a>
+    </footer>
+  </div>
+)
+```
+
+## Styling the news
+
+We've got the basics of our news headlines. Let's make it look incredible. Try experimenting with the CSS now! Add a tag of `<style jsx>{\`your css\`}</style>` and play around. Don’t just copy this example, try your own :)
```
Comment (MaxWofford) [inline workshops/nextjs_dashboard/README.md]: Looks like this markdown doesn't do what you want it to. Copy, pasted:

`<style jsx>{\`your css\`}</style>`

**Comment by MaxWofford** [comment]
@lachlanjc this is great. Just went over the example projects & I'd love to see this merged. You've got my seal of approval once my comments are addressed (don't need to ask before merging)

**Pull request merged**